packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -86,9 +87,58 @@ class _DriverBinding extends BindingBase with SchedulerBinding, ServicesBinding,
 /// for tests where exceptions are expected. Defaults to false. Any errors
 /// will still be returned in the `response` field of the result JSON along
 /// with an `isError` boolean.
-void enableFlutterDriverExtension({ DataHandler handler, bool silenceErrors = false }) {
+///
+/// The `finders` parameter are used to add custom finders, as in the following example.
+///
+/// ```dart main
+/// void main() {
+///   enableFlutterDriverExtension(finders: <FinderExtension>[ SomeFinderExtension() ]);
+///
+///   app.main();
+/// }
+/// ```
+///
+/// ```dart
+/// class Some extends SerializableFinder {
+///   const Some();
+///
+///   @override
+///   String get finderType => 'Some';
+///
+///   @override
+///   Map<String, String> serialize() {
+///     // Serialize Finder
```

Comment:
Please add an example of what should be done here, and perhaps a comment explaining when you can leave this empty.